src/fparser/statements.py

```diff
@@ -641,13 +642,21 @@ class Allocate(Statement):
     match = re.compile(r'allocate\s*\(.*\)\Z', re.I).match
 
     def process_item(self):
+        '''
+        Process the allocate statement and store the various entities being
```

Comment:
For the sake of consistency with other docstrings for this method, please replace "allocate" with "ALLOCATE".

Reply:
Done.